README.md

```diff
@@ -1 +1,6 @@
-# hello-world
\ No newline at end of file
+hello world
+the target of this project is to get sure about the tools that will be used through the course
+
+Built With
+html and css
+
```

Comment:
- [ ] Kindly make your readme file more descriptive by using the [microverse readme template](https://github.com/microverseinc/readme-template) this make your project more descriptive and professional 👍🏿